src/main.cpp

```diff
@@ -339,20 +329,17 @@ int main(int argc, char **argv) {
       msg << " Key Instance: " << std::dec << BSLONG(opt->des.keyInstance)
           << std::endl;
 
-      if (logFile.is_open()) {
-        logFile << timestamp() << ": " << msg.str();
-      }
+      syslog(LOG_NOTICE, "%s", msg.str().c_str());
     } else {
       std::stringstream msg{};
 
       msg << "Encryption turned off for device '" << tapeDrive << "'.";
       msg << " Key Instance: " << std::dec << BSLONG(opt->des.keyInstance)
           << std::endl;
 
-      if (logFile.is_open())
-        logFile << timestamp() << ": " << msg.str();
+      syslog(LOG_NOTICE, "%s", msg.str().c_str());
     }
-    std::cout << "Success! See '" << LOGFILE << "' for a key change audit log."
+    std::cout << "Success! See system logs for a key change audit log."
```

Comment:
can we change `See system logs` with a function to get the path here?
`"Success! See" <<  mystic_systemlogs_function()  << "for a key change audit log."`

Reply:
Not easily since the system can be configured in /etc/syslog.conf to send stenc's messages to some arbitrary destination.